index.html

```diff
@@ -6,35 +6,187 @@
+              <img src="./elements/icn.png" alt="circle" class="icon">
+              <span class="year">2015</span>
+            </div>
+      </div>
+      <p class="crdtxt">A daily selection of privately personalized reads;
+         no accounts or sign-ups required.</p>
+      <ul class="tags">
+        <li class="tgs tgs1">html</li>
+        <li class="tgs tgs2">css</li>
+        <li class="tgs tgs3">javaScript</li>
+      </ul>
+      <div class="actn">
+          <button class="prjctbtn">
+            <span class="btntxt">See Project</span>
+          </button>
+      </div>
+    </div>
+    <div class="card">
+      <img src="./elements/snpsht3.png" alt="Tonic2">
+      <div class="prmry-txt">
+            <span class="title">Tonic</span>
+            <div class="frm-2">
+              <span class="clnt">CANOPY</span>
+              <img src="./elements/icn.png" alt="circle" class="icon">
+              <span class="role">Back End Dev</span>
+              <img src="./elements/icn.png" alt="circle" class="icon">
+              <span class="year">2015</span>
+            </div>
+      </div>
+      <p class="crdtxt">A daily selection of privately personalized reads;
+         no accounts or sign-ups required.</p>
+      <ul class="tags">
+        <li class="tgs tgs1">html</li>
+        <li class="tgs tgs2">css</li>
+        <li class="tgs tgs3">javaScript</li>
+      </ul>
+      <div class="actn">
+          <button class="prjctbtn">
+            <span class="btntxt">See Project</span>
+          </button>
+      </div>
+    </div>
+    <div class="card">
+      <img src="./elements/snpsht4.png" alt="multi-post2">
+      <div class="prmry-txt">
+            <span class="title">Multi-Post <br>Stories</span>
+            <div class="frm-2">
+              <span class="clnt">CANOPY</span>
+              <img src="./elements/icn.png" alt="circle" class="icon">
+              <span class="role">Back End Dev</span>
+              <img src="./elements/icn.png" alt="circle" class="icon">
+              <span class="year">2015</span>
+            </div>
+      </div>
+      <p class="crdtxt">A daily selection of privately personalized reads;
+         no accounts or sign-ups required.</p>
+      <ul class="tags">
+        <li class="tgs tgs1">html</li>
+        <li class="tgs tgs2">css</li>
+        <li class="tgs tgs3">javaScript</li>
+      </ul>
+      <div class="actn">
+          <button class="prjctbtn">
+            <span class="btntxt">See Project</span>
+          </button>
+      </div>
+    </div>
+  </section>
+  <section class="about">
+    <section class="abtmyslf art1">
+      About<br>Myself
+    </section>
+    <p class="abttxt art2">
+      Hello I’m a software developer! I can help you build a product,
+      feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded,
+      don’t hesitate to contact me.
+    </p>
+    <p class="ltscnct">LET’S CONNECT</p>
```

Comment:
Not to be repetitive, but this is also a kind of subtitle, so kindly try to use an: <details><summary> `<hx>`</summary>
    <pre>
    - `<h3>`
    - `<h4>`
    - `<h5>`
    </pre>
   </details> for this 😃

Reply:
- [ ] For the same reasons, it would be nice if you implement the same HTML subtitle `hx` tag here.